docs/getting-started-developers.md

```diff
@@ -36,7 +36,7 @@ The client lives in the `src/client` directory. To run the client for developmen
 yarn start
 ```
 
-to begin development. The client was bootstrapped using [Create-React-App with TypeScript](https://facebook.github.io/create-react-app/docs/adding-typescript).
+to begin development in the browser. The client was bootstrapped using [Create-React-App with TypeScript](https://facebook.github.io/create-react-app/docs/adding-typescript).
```

Comment:
Should we mention to run on a chromium based browser I.e Anaheim/Chrome. Not sure if it works on edge (Not anaheim), remember there were some issues.